network/src/commonMain/kotlin/com/wire/kalium/network/api/notification/pushToken/PushTokenRequestBody.kt

```diff
@@ -0,0 +1,17 @@
+package com.wire.kalium.network.api.notification.pushToken
+
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class PushTokenRequestBody(
+    @SerialName("app")
+    val senderId: String,
+    @SerialName("client")
+    val client: String,
```

Comment:
The clientID is optional 
```suggestion
    val client: String?,
```

Reply:
actually it's not , the swagger need an update for this